examples/src/main/python/als.py

```diff
@@ -21,8 +21,6 @@
 
 This example requires numpy (http://www.numpy.org/)
 """
-from __future__ import print_function
-
```

Comment:
Let's don't do this for now ..  we should officially drop Python 2 first.

Reply:
Oh, I see. I guess it's probably more than 6 months away? I can close the two prs if that's the case.

Reply:
Nobody knows but at least that wouldn't happen until Spark 3.0 release which will take a couple of months at least. Let's leave it closed for now.